src/plugins/pgcomparator_cdc_operator.py

```diff
@@ -85,14 +88,17 @@ def execute(self, context: Dict[str, Any]) -> None:
             # first query parameter and has no value.
             source_url += f"?{self.key_column}"
         target_url = f"{db_url}/{self.target_table}"
-
+        program_path = first_true(
+            (Path("/usr/bin/pg_comparator"), Path("/usr/local/bin/pg_comparator")),
+            pred=lambda p: p.exists(),
+        )
         self.log.info(
-            "Start change data capture (with pg_comparator) from: '%s' to: '%s" ".",
+            "Start change data capture (with pg_comparator) from: '%s' to: '%s.",
             source_url,
             target_url,
         )
         arguments = [
-            "/usr/bin/pg_comparator",
+            program_path.as_posix(),
```

Comment:
Looking at the git history, this was oringinally a command without a path. There should have been a reason to change it.

Reply:
@jjmurre I guess FreeBSD came into play here with `/usr/local/bin` :)

Reply:
@jjmurre I'm stupid. I introduced the path component in 84a507d7e7734eaf397e6037d418dab2fda6f169 and didn't document the need for the change. Will revert it back to being pathless as that should work equally well.

@ngaranko Yep :-)

Reply:
Turns out I am not stupid. Or at least, less stupid than I'd thought. In commit 84a507d7e7734eaf397e6037d418dab2fda6f169 I ditched `shell=True` in the `subprocess.run` call. Hence the need for the full path to the executable. Can't rely on a shell to inspect the PATH environment variable as we are not using the shell anymore.

Reply:
`subprocess.run` checks the `PATH` even when `shell=False`. Try `subprocess.run(["ls"], shell=False)`.

Reply:
Ah, didn't know that. Will fix in next PR. Thank you @lbam . Learned something new.